src/Symfony/Component/Translation/DependencyInjection/TranslationLoggingPass.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Translation\TranslatorBagInterface;
+
+/**
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+class TranslationLoggingPass implements CompilerPassInterface
```

Comment:
Keeping it named `LoggingTranslatorPass` would be more consistent with `Symfony\Component\Translation\LoggingTranslator`

Reply:
I tried to be consistant with other CompilerPass that were already in `Translation/DependencyInjection`, I can move back to the old name if preferred .